app/auth/create.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from 'react'
+import { Text, TextInput, HelperText, Button } from 'react-native-paper'
+import { useRouter } from 'expo-router'
+import { useDispatch, useSelector } from 'react-redux'
+
+import AuthPage from '@components/AuthPage'
+import { createUser, selectIsLoggedIn } from '@store'
+
+const CreateAccount = () => {
+    const dispatch = useDispatch()
+    const router = useRouter()
+
+    const [fName, setFName] = useState('')
+    const [lName, setLName] = useState('')
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+    const [passwordConfirm, setPasswordConfirm] = useState('')
+    const [badEmail, setBadEmail] = useState(false)
+    const [passwordsMatch, setPasswordsMatch] = useState(true)
+
+    useEffect(() => {
+        const re = /^[\w-\\.]+@crimson.ua.edu$/
+        if (email.length > 0 && !re.test(email)) {
+            setBadEmail(true)
+        } else {
+            setBadEmail(false)
+        }
+    }, [email])
+
+    useEffect(() => {
+        if (passwordConfirm.length > 0 && passwordConfirm == password) {
```

Comment:
The abstract equality operator (`==`) here shouldn't be an issue but let's change this to the strict equality operator (`===`) since that's what you should almost always use. For more info: https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons